src/main/views/sso-user-successful.njk

```diff
@@ -0,0 +1,39 @@
+{% from "macros/breadcrumb.njk" import breadcrumb %}
```

Comment:
The name of this file should be 'remove-user-sso-successful.njk' stating the action performed. 'sso-user-successful' sounds like they successfully use SSO

Reply:
fixed